chanacceptor/rpcacceptor.go

```diff
@@ -232,9 +246,27 @@ func (r *RPCAcceptor) sendAcceptRequests(errChan chan error,
 				continue
 			}
 
+			var (
+				accept    = resp.Accept
+				acceptErr error
+			)
+
+			if resp.Error != "" {
+				acceptErr = errors.New(resp.Error)
+			}
+
+			// Validate the response we have received. If it is not
+			// valid, we log our error and reject the channel.
+			if err := validateAcceptorResponse(resp); err != nil {
```

Comment:
style nit: perhaps it would simplify this block to have `validateAcceptorResponse` return the final `accept` and `acceptErr` values? something like:
```golang
accept, acceptErr, err := validateAcceptorResponse(resp)
if err != nil {
     log...
}
```